docs/guide/deployment.md

```diff
@@ -319,3 +319,90 @@ Verify your project is successfully published by Surge by visiting `myawesomepro
 
     cd -
     ```
+
+
+### Docker (Nginx)
+
+Deploying your application using nginx inside of docker can be done easily with a minor modification of the nginx configuration.
+
+1. Install [docker](https://www.docker.com/get-started)
+
+2. Create a `Dockerfile` file in the root of your project.
+
+    ```Dockerfile
+    FROM node:10
+    COPY ./ /app
+    WORKDIR /app
+    RUN npm install && npm run build
+
+    FROM nginx
+    RUN mkdir /app
+    COPY --from=0 /app/dist /app
+    COPY nginx.conf /etc/nginx/nginx.conf
+    ```
+
+3. Create a `.dockerignore` file in the root of your project
+
+    Setting up the `.dockerignore` file correctly prevents `node_modules` and any intermediate build artifacts from being copied to the image which can cause issues during building.
+
+    ```gitignore
+    **/node_modules
+    **/dist
+    ```
+
+4. Create a `nginx.conf` file in the root of your project
+
+    This is a simple `nginx` configuration that falls back to the root `index.html` page for `404's`. This is necessary when using `pushState()` based routing.
```

Comment:
It can be reworded to:

Following `nginx` configuration serves root `index.html` for `page not found` errors.

Reply:
Done